astro/src/content/docs/extend/code/lambdas/lambda-remote-api-calls.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Making API Calls From Lambdas
+description: An overview of how to make API calls from Lambdas.
+section: extend
+subcategory: code
+tertcategory: lambdas
+topOfNav: true
+---
+import AdvancedPlanBlurb from 'src/content/docs/_shared/_advanced-plan-blurb.astro';
+import APIBlock from 'src/components/api/APIBlock.astro';
+import APIField from 'src/components/api/APIField.astro';
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+import LambdaTypes from 'src/content/docs/_shared/_lambda-types.astro';
+import MembershipLambda from 'src/content/docs/extend/code/_membership-lambda.md';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+<AdvancedPlanBlurb />
+
+## Overview
+
+Lambda HTTP Connect allows you to make HTTP requests from within a lambda. Any lambda can make a request to any network accessible URL.
+
+This features allows you to access data from external systems to configure token claims or to add data to a user profile. It also allows you to push profile or other data from FusionAuth to an external system as needed.
+
+Here's a video showing more details about Lambda HTTP Connect:
+
+<YouTube id="_TnDUPQm3aQ" />
+
+### Example Lambda
```

Comment:
```suggestion
## Example Lambda
```
I think this section (and following ones) should be at the same level as Overview